rr/gstreamer/gst_media.py

```diff
@@ -198,13 +250,21 @@ def make(cls, desc, all_triggers):
 
 
 class GstImage():
-    def __init__(self, width, height, format, sample, gst_media_obj):
+    def __init__(
+            self,
+            tensor_data_layout,
```

Comment:
This will break a lot of tests. Please check tests before submitting PRs. I would avoid extending the image with this. You're not using for anything.